.github/workflows/release.yml

```diff
@@ -79,14 +79,18 @@ jobs:
           java-version: 1.8
           server-username: MAVEN_USERNAME
           server-password: MAVEN_PASSWORD
+      - id: install-secret-key
+        name: Install gpg secret key
+        run: |
+          cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
```

Comment:
I suppose these keys remain installed after the build is over... Do we wipe build hosts after publishing?

Reply:
I'm not entirely sure what GH does with their infra. I'm assuming after the build is complete they tear everything down.

Reply:
And the number of secrets the actions are already passed I think gpg key is the least of our problems :)